src/process.c

```diff
@@ -180,3 +180,7 @@ void process_dump(struct process *p) {
     console_printf("esp: %x\n", s->esp);
     console_printf("eip: %x\n", s->eip);
 }
+
+void __process_set_initial_process_ready(struct process *p) {
```

Comment:
why are we switching to the __ naming convention here? We haven't used it anywhere else